examples/nidigitalpattern/configure-voltage-levels-and-termination-voltage.py

```diff
@@ -1,56 +1,64 @@
-# Link :  https://www.ni.com/en-in/support/downloads/drivers/download.ni-digital-pattern-driver.html#367315
-#
-# For instructions on how to use protoc to generate gRPC client interfaces, see
-# our "Creating a gRPC Client" wiki page.
-# Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
-#
-# Refer to the NI-Digital Pattern Driver gRPC Wiki for the latest C Function Reference:
-# Link: https://github.com/ni/grpc-device/wiki/NI-DIGITAL-PATTERN-DRIVER-C-Function-Reference
-#
-# Running from command line:
-# Server machine's IP address, port number, and resource name can be passed as separate command line arguments.
-#   > python configure-voltage-levels-and-termination-voltage.py <server_address> <port_number> <resource>
-# If they are not passed in as command line arguments, then by default the server address will be "localhost:31763"
-import grpc
-import sys
+r"""Create an instrument session, configure various voltage levels, and then burst a pattern.
+
+The vol, voh, vih, vil, iol, ioh and vcom values are configurable parameters and the user can select
+from High-Z (h), Active Load (a) and Three-Level Drive (t) termination options in term_select.
+
+The gRPC API is built from the C API. NI-Digital Pattern Driver Pattern documentation is installed
+with the driver at:
+  C:\Users\Public\Documents\National Instruments\NI-Digital-Pattern-Driver\Documentation\Digital Pattern Help.chm
+
+Copy the .pinmap, .specs, .digitiming & .digipat files that come with this example to a folder on
+the server machine & copy-paste the path of the folder to the directory_path variable below.
+
+Use the NI Digital Pattern Editor to create or modify pin or channel map files:
+  https://www.ni.com/documentation/en/ni-digital/20.6/digital-pattern-editor/pin-channel-map-editor/
+
+Getting Started:
+
+To run this example, install "NI-Digital Pattern Driver" on the server machine:
+  https://www.ni.com/en-in/support/downloads/drivers/download.ni-digital-pattern-driver.html#367315
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Refer to the NI-Digital Pattern Driver gRPC Wiki for the latest C Function Reference:
+  https://github.com/ni/grpc-device/wiki/NI-DIGITAL-PATTERN-DRIVER-C-Function-Reference
+
+Running from command line:
+
+Server machine's IP address, port number, and resource name can be passed as separate command line
+arguments.
+  > python configure-voltage-levels-and-termination-voltage.py <server_address> <port_number> <resource>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763", with "PXI1Slot2" as the resource name.
+"""  # noqa: W505
+
 import os
+import sys
+
+import grpc
 import nidigitalpattern_pb2 as nidigital_types
 import nidigitalpattern_pb2_grpc as grpc_nidigital
 
 server_address = "localhost"
 server_port = "31763"
```

Comment:
Because these variables are module scoped, aren't we allowed for them to be upper snake case to "indicate" they are constants

Reply:
Yes, and they should be, but the linter didn't point those out. I'll fix them either in this PR or a follow-up.